src/Application.php

```diff
@@ -472,8 +309,14 @@ public function route($path, $middleware = null, array $methods = null, $name =
         $this->checkForDuplicateRoute($path, $methods);
 
         if (! isset($route)) {
-            $methods = (null === $methods) ? Router\Route::HTTP_METHOD_ANY : $methods;
-            $route   = new Router\Route($path, $middleware, $methods, $name);
+            $methods    = (null === $methods) ? Router\Route::HTTP_METHOD_ANY : $methods;
```

Comment:
Yes; empty array means "responds to only implicit methods". It was due to a feature request somewhere along the line.